tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -820,10 +820,10 @@ class ExecutionTest {
         return false;
       }
 
-      if (GetModuleHandle("d3d10warp.dll") != NULL) {
-        CHAR szFullModuleFilePath[MAX_PATH] = "";
-        GetModuleFileName(GetModuleHandle("d3d10warp.dll"),
-                          szFullModuleFilePath, sizeof(szFullModuleFilePath));
+      if (GetModuleHandleW(L"d3d10warp.dll") != NULL) {
```

Comment:
This change looks good. However, I've seen projects where the use of the W api function vs the char one was controlled via a compile time flag. Would there be any reason to do that here? 

This is just a test case so i'm assuming no. But was curious what if we wanted to cross compile a windows build on linux? what if we wanted to use wine  to run test cases or some other windows emulation layer? Do WCHARs preclude us from this?

Reply:
By the way, this is the reason for all the forced conversions in the DXC API.  The only Unicode API style for Windows (and thus COM APIs) is the wide string APIs, yet we internally use char strings representing Unicode in UTF-8 encoding.